src/core/email/templates/progress-report-status-changed.template.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Button,
+  Column,
+  HideInText,
+  Section,
+  Text,
+} from '@seedcompany/nestjs-email/templates';
+import { startCase } from 'lodash';
+import { DateTime } from 'luxon';
+import { EmailReportStatusNotification } from '../../../components/progress-report/workflow/progress-report-workflow.service';
+import { fullName } from '../../../components/user/fullName';
+import { EmailTemplate, Heading } from './base';
+import { useFrontendUrl } from './frontend-url';
+
+export function ProgressReportStatusChanged({
+  changedBy,
+  project,
+  languageEngagement,
+  report,
+  reportQtrLabel,
+  newStatusVal,
+  previousStatusVal,
+}: EmailReportStatusNotification) {
+  const projectUrl = project?.id
+    ? useFrontendUrl(`/projects/${project.id}`)
+    : useFrontendUrl(`/projects/no-project-found`);
+  const projectName = project?.name.value;
+
+  const reportUrl = report.id
+    ? useFrontendUrl(`/progress-reports/${report.id}`)
+    : useFrontendUrl(`/projects/no-project-reports-found`);
+  const reportLabel = reportQtrLabel
+    ? `Quarterly Report - ${reportQtrLabel}`
+    : 'Quarterly Report';
+
+  const changerUrl = useFrontendUrl(`/users/${changedBy.id}`);
+  const changerName = fullName(changedBy);
+
+  const oldStatus = startCase(previousStatusVal) || undefined;
+  const newStatus = startCase(newStatusVal) || undefined;
+
+  const changedAt = DateTime.now()
+    .setZone(changedBy.timezone.value ?? 'America/Chicago')
```

Comment:
It would be better to inject this default value with a hook context.
```suggestion
    .setZone(changedBy.timezone.value ?? useDefaultTimezone())
```

Just like we do for frontend urls.
https://github.com/SeedCompany/cord-api-v3/blob/master/src/core/email/templates/frontend-url.tsx

https://github.com/SeedCompany/cord-api-v3/blob/cc00184941ece5480fc301c9bfbec308e27dac93/src/core/config/config.service.ts#L105-L105
https://github.com/SeedCompany/cord-api-v3/blob/cc00184941ece5480fc301c9bfbec308e27dac93/src/core/config/config.service.ts#L120-L122